src/v/kafka/requests/requests.cc

```diff
@@ -38,6 +38,8 @@
 #include <seastar/core/print.hh>
 #include <seastar/util/log.hh>
 
+#include <fmt/ostream.h>
```

Comment:
do you need this?

the rest makes sense.

Reply:
Fixed